common/src/main/java/org/tron/core/Constant.java

```diff
@@ -20,6 +20,12 @@
 
   public static final String DATABASE_DIR = "storage.directory";
 
+  // locate in storageDbDirectory, store the db infos,
+  // now only has the snapshot block number
+  public static final String INFO_FILE_NAME = "info.properties";
+  // the block number that split between the snapshot and history
+  public static final String SPLIT_BLOCK_NUM = "split_block_num";
```

Comment:
mixed use of `snake_case` and `camelCase` keys.

Reply:
The naming rule of constant in java is just this